api_sentry.py

```diff
@@ -0,0 +1,201 @@
+import os
+import sys
+
+import pandas as pd
+
+from lib.sentry_conn import APIClient
+
+from database import (
+    Database,
+    ReportSentryIssues
+)
+
+# The 2 major versions are beta and release.
+NUM_MAJOR_VERSIONS = 2
+
+
+class Sentry:
+
+    def __init__(self):
+        try:
+            self.client = APIClient(os.environ['SENTRY_HOST'])
+            self.client.api_token = os.environ['SENTRY_API_TOKEN']
+            self.organization_slug = \
```

Comment:
Is this different than project_slug in line 26?

Reply:
No, the slugs are the same. Removed project_slug.